### PR TITLE
cce node add k8s_tags and user_tags

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
fixes: #183

### CODE REVIEW

File: flexibleengine/resource_flexibleengine_cce_node_v3.go [hunk 1]
```diff
@@ -61,6 +62,16 @@ func resourceCCENodeV3() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"user_tags": {
```
Comment (niuzhenguo) [inline flexibleengine/resource_flexibleengine_cce_node_v3.go:65]: better to just name it to tags, and should be conflict with lables: ConflictsWith: []string{"labels"} and deprecated it.

Comment (Jason-Zhang9309) [reply flexibleengine/resource_flexibleengine_cce_node_v3.go]: fixed

File: flexibleengine/resource_flexibleengine_cce_node_v3.go [hunk 2]
```diff
@@ -57,6 +58,19 @@ func resourceCCENodeV3() *schema.Resource {
...
+				Type:          schema.TypeMap,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"tags"},
+				Deprecated:    "use tags instead",
+			},
+			"tags": {
+				Type:          schema.TypeMap,
+				Optional:      true,
+				ForceNew:      true,
```
Comment (niuzhenguo) [inline flexibleengine/resource_flexibleengine_cce_node_v3.go:70]: should be updatable.

**Comment by osaluden** [comment]
Hello
I'm a little bit concerned by the comment in the code regarding the labels: "Deprecated:    "use tags instead"". 
I disagree with that since "labels" is a Kubernetes parameters in order to put a label on the Kubernetes node and use them as a Pod placement constraint,  whereas "tag" is a Flexible Engine parameter.
Both must be available.
Did I misunderstood your change ?
B.R.
Olivier

**Comment by niuzhenguo** [comment]
@osaluden I understand your concerns. 
So you would like to keep `labels` instead of adding a new `k8s_tags` which puts a label on the Kubernetes node.
Adds `tags` like what we do to make them show on the Flexible Engine Console.

**Comment by osaluden** [comment]
@niuzhenguo Yes indeed. 
I want to keep both because these are two independant parameters, one for K8S (labels) and one for FE (tags)
Thanks you for yo rprompt reply!
Olivier

**Comment by niuzhenguo** [comment]
@osaluden Thanks for your suggestions!
@Jason-Zhang9309 Please update the parameters as @osaluden suggested above, thanks!

**Comment by Jason-Zhang9309** [comment]
@osaluden @niuzhenguo These two parameters have been updated as @osaluden suggested.

**Comment by osaluden** [comment]
Thank you @niuzhenguo  and @Jason-Zhang9309  !

**Pull request merged**